addons/website_sale/static/src/js/location_selector/location_schedule/location_schedule.js

```diff
@@ -0,0 +1,11 @@
+import {
+    LocationSchedule
+} from '@delivery/js/location_selector/location_schedule/location_schedule';
+import { patch } from '@web/core/utils/patch';
+import { _t } from '@web/core/l10n/translation';
+
+patch(LocationSchedule.prototype, {
+    get closedLabel() {
+        return _t("Closed");
+    },
```

Comment:
Isn't there any other way than duplicating this method? Otherwise it means to edit both files everytime a change is needed.
If it is the case, I'd add a comment about it.